README.md

```diff
@@ -89,6 +89,53 @@ Basic proof of 'ipfs working' locally:
 	ipfs cat <that hash>
 
 
+### Docker usage
+
+An ipfs docker image is hosted at [hub.docker.com/u/jbenet/go-ipfs](hub.docker.com/u/jbenet/go-ipfs).
```

Comment:
sorry, this image was outdated. I've updated that repo to point to https://github.com/ipfs/go-ipfs

(i wish i could get the dockerhub repo to be https://registry.hub.docker.com/repos/ipfs/ but someone owns it i think :/)